lib/components/overlays/quick_settings/quick_settings_overlay.dart

```diff
@@ -117,6 +124,49 @@ class _QuickSettingsOverlayState extends State<QuickSettingsOverlay>
   }
 }
 
+PageRouteBuilder customRouteTransition(Widget screen) {
+  return PageRouteBuilder(
+    transitionDuration: Constants.animationDuration,
+    reverseTransitionDuration: Constants.animationDuration,
+    pageBuilder: (context, animation, secondaryAnimation) => screen,
+    transitionsBuilder: (context, animation, secondaryAnimation, child) {
+      final fadeAnimationIn = Tween(
+        begin: animation.value,
+        end: animation.value,
+      ).animate(animation);
+
+      final fadeAnimationOut = Tween(
+        begin: animation.value,
+        end: 1 - animation.value,
+      ).animate(secondaryAnimation);
```

Comment:
`final fadeAnimationOut = Tween(begin: 1.0, end: 0.0).animate(secondaryAnimation);`

As before as i didn't realize you needed Animation<double> and not double

Reply:
This works for the same reason as what I wrote in the review below, except that since it's triggered in reverse, it goes from 1 to 0. Changed

Reply:
That is barely understandable as the other one plus mixing animation and secondaryAnimation is a recipe for disaster

Reply:
Ye I agree, done